vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/experiments/experiment/ExperimentInformationDialog.java

```diff
@@ -28,139 +38,266 @@ public class ExperimentInformationDialog extends DialogWindow {
 
   @Serial
   private static final long serialVersionUID = 2142928219461555700L;
-  private static final String TITLE = "Experimental Design";
-  private final DefineExperimentComponent defineExperimentComponent;
-  private final List<ComponentEventListener<CancelEvent<ExperimentInformationDialog>>> cancelEventListeners = new ArrayList<>();
-  private final List<ComponentEventListener<ConfirmEvent<ExperimentInformationDialog>>> confirmEventListeners = new ArrayList<>();
-  private final MODE mode;
+
+  private static final String CHIP_BADGE = "chip-badge";
+  private static final String WIDTH_INPUT = "full-width-input";
```

Comment:
NIT: I don't think it's necessary to split this up into 2 distinct css classes but rather define the multiselectioncombobox css accordingly in the dialog itself